### PR TITLE
feat: add frontend e2e report on pr comment

### PR DESCRIPTION
**Pull request by LiteSun**
Please answer these questions before submitting a pull request

- Why submit this pull request?
- [ ] Bugfix
- [x] New feature provided
- [ ] Improve performance
- [ ] Backport patches

- Related issues

___
### Bugfix
- Description

- How to fix?

___
### New feature or improvement
- Describe the details and related test reports.

___
### Please add the corresponding test cases if necessary.

___
### Backport patches
- Why need to backport?

- Source branch

- Related commits and pull requests

- Target branch

### CODE REVIEW

File: .github/workflows/backend-unit-test.yml [hunk 1]
```diff
@@ -36,7 +36,7 @@ jobs:
       - name: upload coverage profile
         working-directory: ./api
         run: |
-          bash <(curl -s https://codecov.io/bash)
+          bash <(curl -s https://codecov.io/bash) -F backend-unit-test
```
Comment (starsz) [inline .github/workflows/backend-unit-test.yml:39]: Only want to know why this line doesn't need "-f" option.

Comment (LiteSun) [reply .github/workflows/backend-unit-test.yml:39]: ping @nic-chen

Comment (nic-chen) [reply .github/workflows/backend-unit-test.yml:39]: @LiteSun  we could use `-F backend` for all backend test

Comment (LiteSun) [reply .github/workflows/backend-unit-test.yml:39]: I think the back-end unit test and e2e test coverage should be separated, if you want to see the overall back-end coverage, please see https://app.codecov.io/gh/apache/apisix-dashboard/compare/1529/tree.
image

 What do you think? @membphis @juzhiyuan

**Review by nic-chen — COMMENTED** [review]
need to add `backend-e2e-test`

**Review by juzhiyuan — APPROVED** [review]

**Review by nic-chen — APPROVED** [review]

**Pull request merged**